tango_streamer/Rf_stamped indata
---
string[2] outmsg
